app/src/main/java/com/vci/vectorcamapp/ui/theme/Colors.kt

```diff
@@ -32,6 +32,9 @@ data class Colors(
     val cardBackground: Color = Color(0xFFFFFFFF),
     val appBackground: Color = Color(0xFFF7F8F7),
     val transparent: Color = Color(0x00000000),
+    val animationScanColor: Color = Color(0xFF00FF7F),
+    val animationOverlayColor: Color = Color(0x80000000),
+    val animationCornerColor: Color = Color(0xFFFFFFFF)
 )
```

Comment:
The purpose of the colors theme is to have reusable colors. I would recommend not having colors for individual elements because then our list of colors will keep growing indefinitely. I would recommend using one of the colors already available in the pallete, or defining a generic color name which you think will be used for multiple components in the future.